docs/general-usage.md

```diff
@@ -73,7 +73,7 @@ For example:
     AutoStreaming = "false"
 ```
 <p align="left">
-      <i>Sample: Snippet from general.usb.camera.toml.example</i>
+      <i>Sample: Snippet from general.usb.camera.yaml.example</i>
```

Comment:
As with ONVIF, the above in-line TOML example must be converted to YAML. Please make sure this is done for all in-line TOML.

Reply:
Yep I have already converted those example files to Yaml, both in the name of the file and the formatting of the file

Reply:
@marcpfuller , not the files. The in-line TOML in the doc files.